src/test/ui/proc-macro/auxiliary/custom-quote.rs

```diff
@@ -7,7 +7,7 @@
 
 extern crate proc_macro;
 use std::str::FromStr;
-use proc_macro::*;
+use proc_macro::{Delimiter, Group, TokenStream, TokenTree};
```

Comment:
To answer the inevitable question, this is necessary because the new `MultiSpan` trait has a `.span()` method, which would override the inherent method when both are in scope.